tools/calculate-release/CalculateRelease.hs

```diff
@@ -48,8 +48,8 @@ main = do
 
     putStrLn $ "Mainnet fork height (at " <> show mainnetForkTime <> "): " <> show mainnetForkHeight
     putStrLn $ "Testnet fork height (at " <> show testnetForkTime <> "): " <> show testnetForkHeight
-    let nextServiceDateDay = addGregorianMonthsClip 4 serviceDateDay
-    putStrLn $ "Next service date (+4 months): " <> show nextServiceDateDay
+    let nextServiceDateDay = addGregorianMonthsClip 2 serviceDateDay
+    putStrLn $ "Next service date (+2 months): " <> show nextServiceDateDay
```

Comment:
We used to do service releases on Thursdays 00:00:00 UTC. Doing it in terms of month (and not weeks) could result in releases being on weekends or during holidays. Is this intended?

Reply:
We definitely want to keep to Thursdays at 0 UTC, for sanity's sake.